samples/ar-view-node/src/test/java/com/example/sceneviewsample/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.sceneviewsample
```

Comment:
Should we remove the test files and libraries if they aren't used?

Reply:
removed